build.go

```diff
@@ -105,15 +162,27 @@ func Build(procs ProcMgr, logger scribe.Emitter) packit.BuildFunc {
 			return packit.BuildResult{}, fmt.Errorf("failed to copy procmgr-binary into layer: %w", err)
 		}
 
-		processes := []packit.Process{
-			{
-				Type:    "web",
-				Command: "procmgr-binary",
-				Args:    []string{filepath.Join(layer.Path, "procs.yml")},
-				Default: true,
-				Direct:  true,
-			},
+		originalProcess := packit.Process{
+			Type:    "web",
+			Command: "procmgr-binary",
+			Args:    []string{filepath.Join(layer.Path, "procs.yml")},
+			Default: true,
+			Direct:  true,
 		}
+
+		processes := make([]packit.Process, 0)
+
+		if _, err := reloader.ShouldEnableLiveReload(); err != nil {
+			return packit.BuildResult{}, err
+			//} else if shouldEnableReload {
+			//	nonReloadableProcess, reloadableProcess := reloader.TransformReloadableProcesses(originalProcess, libreload.ReloadableProcessSpec{
+			//		WatchPaths: watchPaths,
+			//	})
+			//	processes = append(processes, nonReloadableProcess, reloadableProcess)
```

Comment:
Can we delete this?

Reply:
But also, where are the watch paths used in the code? I see where they're getting updated but not used.

Reply:
oops I think this is an out of date comment